internal/template-validator/tlsinfo/tlsinfo.go

```diff
@@ -174,31 +162,80 @@ func loadCertificates(directory string) (serverCrt *tls.Certificate, err error)
 	return &cert, nil
 }
 
-func (ti *TLSInfo) getCertificate() *tls.Certificate {
-	ti.certLock.Lock()
-	defer ti.certLock.Unlock()
-	return ti.cert
+func (ti *TLSInfo) updateTLSOptions(directory string) {
+	tlsOptions, err := loadTLSOptions(directory)
+
+	ti.lock.Lock()
+	defer ti.lock.Unlock()
+
+	if err != nil {
+		ti.minTLSVersion = 0
+		ti.cipherSuites = nil
+		ti.tlsOptionsError = err
+		logger.Log.Info("failed to TLS options",
```

Comment:
Is it an error of level Info? Shouldn't it be higher?

Reply:
Also, what did it fail to do? :)

Reply:
Yes, changed it to error.